app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="icon_background">#1D70B8</color>
```

Comment:
As this value doesn't change between default and night I think you can safely omit it